webclient/src/types/server.tsx

```diff
@@ -93,11 +99,13 @@ export const getHostPort = (host: Host): { host: string, port: string } => {
 };
 
 export enum KnownHost {
+  CHICKATRICE = 'Chickatrice',
```

Comment:
Fairly certain these are no longer used, if you can be bothered, you can remove this enum and the below const entirely, and just make sure the build still runs fine.